scripts/pre_commit_linter.py

```diff
@@ -3203,4 +3385,8 @@ def main(args=None):
 
 
 if __name__ == '__main__':
+    NAME_SPACE = multiprocessing.Manager().Namespace()
+    PROCESSES = multiprocessing.Manager().dict()
+    NAME_SPACE.files = FileCache()
+    FILE_CACHE = NAME_SPACE.files
```

Comment:
@cuichenli, run_presubmit_checks calls pre_commit_linter.main() and in that case `FILE_CACHE` will not be defined, right? Resulting in an error #7704 (not sure!)

Reply:
Yes, that is correct, but it should be safe to move it out of this if statement. If you want I can raise a PR for this.